src/main/java/com/redhat/cloud/notifications/models/Application.java

```diff
@@ -31,6 +33,11 @@
     @JsonInclude(JsonInclude.Include.NON_NULL)
     private Set<EventType> eventTypes; // optional
 
+    @NotNull
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    @JsonProperty("bundle_id")
+    private UUID bundleId;
```

Comment:
This one is marked as non null and at least 1 API call doesn't return it. (It breaks a validation in the openapi.json)

Reply:
Good catch and thanks for the PR. I need to improve the tests.